pymongo/cursor.py

```diff
@@ -207,6 +207,7 @@ def __init__(
         if session:
             self.__session = session
             self.__explicit_session = True
+            self.__session._start_serv_sess()
```

Comment:
Shouldn't this call be deferred to when the session is actually used (in apply_to)?

Reply:
Done.